src/Discord.py

```diff
@@ -0,0 +1,69 @@
+import requests, json
+from src.Dev import Dev
+from src.Settings import Settings
+from src.TERMGUI.Log import Log
+
+from discord_webhook import DiscordWebhook
```

Comment:
@vindennl48 I took my best guess at formatting and style but I'm way out of my element here. Please feel free to make suggestions or tweaks.